app/views/posts/_list_item.haml

```diff
@@ -53,7 +53,7 @@
     by
     = user_mem_link(post.last_user_id, post.last_user_name, post.last_user_deleted?)
   - if local_assigns[:check_box_name]
-    %td.post-check-box{class: klass}= check_box_tag check_box_name, post.id, false, class: 'checkbox'
+    %td.post-check-box{class: klass}= check_box_tag check_box_name, post.id, false, class: 'checkbox', id: "#{check_box_name.delete('[]')}-#{post.id}"
```

Comment:
Do we use this ID? It's a little weird that we're naming it, for example, `marked_ids_9`, if we don't actually need an ID, so maybe it should have `id: nil`?